### PR TITLE
Installation using ansible

### PR DESCRIPTION
**Pull request by ravisaxena23**
**What type of PR is this?**
> /kind documentation

**What this PR does / why we need it**:
this PR add the documentation for installing local cluster using ansible. we are going to remove wiki hence we need it.

**Test Report Added?**:
> /kind NOT-TESTED

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ravisaxena23** [comment]
@skdwriting @anvithks

**Comment by ravisaxena23** [comment]
this is the same pr as the older one. sorry for the new pr. in old pr I made a blunder. I tagged all the new changes so I have to make a new PR. I have closed the previous one.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+```bash
+curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+```
+<br /> Install golang
+
+```bash
+wget https://storage.googleapis.com/golang/go1.13.0.linux-amd64.tar.gz
+tar -C /usr/local -xzf go1.12.1.linux-amd64.tar.gz
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:36]: Incorrect version of Golang.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+<br /> Install docker-compose:
+
+```bash
+curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+```
+<br /> Install golang
+
+```bash
+wget https://storage.googleapis.com/golang/go1.13.0.linux-amd64.tar.gz
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:35]: Please check if this is the correct version. I think the latest stable version is
[https://dl.google.com/go/go1.13.12.linux-amd64.tar.gz](https://dl.google.com/go/go1.13.12.linux-amd64.tar.gz)

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+<strong>NOTE</Strong> : Please ensure that you are using hotpot version >= v0.6.1.
+<br />
+
+Update the file `ansible/group_vars/telemetry.yml` and change the value of enable_telemetry_tools to true
+
+```bash
+# Do you need to install or clean up telemetry tools?
+enable_telemetry_tools: false
+```
+<br /> How to enable Orchestration installation
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:149]: nit: These are subheadings. Please make them #### or ### depending on the nesting.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+
+<br />
+
+Update the file `ansible/group_vars/orchestration.yml` and change the value of `enable_orchestration` to true
+
+```bash
+# Install Orchestration Manager (true/false)
+enable_orchestration: false
+```
+<br /> Set HOST_IP environment variable
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:159]: nit: These are subheadings. Please make them #### or ### depending on the nesting.

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 2 above)
Comment (skdwriting) [reply content/soda-gettingstarted/installation-using-ansible.md:35]: raised #74

**Pull request merged**